tests/templates/kuttl/mount-dags-pvc/health.py

```diff
@@ -5,11 +5,13 @@
 
 if __name__ == "__main__":
     result = 0
+
     log_level = 'DEBUG'
     logging.basicConfig(level=log_level, format='%(asctime)s %(levelname)s: %(message)s', stream=sys.stdout)
 
     http_code = requests.get("http://airflow-webserver-default:8080/api/v1/health").status_code
```

Comment:
Is there a reason why the changed code with the retry loop from the other health.py file hasn't been done here?

Reply:
Changed as per comment

Reply:
If there is a good reason I have no issue whatsoever with keeping it as it is, just wanted to make sure if wasn't an oversight.